spec/support/api/crud.rb

```diff
@@ -20,6 +20,13 @@
   include_context "resource"
   include_context 'resource save'
 
+  # More info about this: https://github.com/3scale/porta/pull/3062#issuecomment-1251161154
```

Comment:
Would be better to add a line like 
> granularity of timestamps is 1 second but we need `updated_at` to be different than `created_at`
> see more at https://github.com/3scale/porta/pull/3062#issuecomment-1251161154

Or whatever wording you like more. In this way the critical information does not rely on an external source. And it is clear why exactly we have this here instead of some long post elsewhere.

Reply:
Done